src/node/chainstatemanager_args.cpp

```diff
@@ -32,13 +32,20 @@ util::Result<void> ApplyArgsManOptions(const ArgsManager& args, ChainstateManage
     if (auto value{args.GetBoolArg("-checkpoints")}) opts.checkpoints_enabled = *value;
 
     if (auto value{args.GetArg("-minimumchainwork")}) {
-        if (!IsHexNumber(*value)) {
-            return util::Error{strprintf(Untranslated("Invalid non-hex (%s) minimum chain work value specified"), *value)};
+        if (auto min_work{uint256::FromUserHex(*value)}) {
+            opts.minimum_chain_work = UintToArith256(*min_work);
+        } else {
+            return util::Error{strprintf(Untranslated("Invalid minimum chain work value specified (%s), must be up to %d characters hex"), *value, uint256::size() * 2)};
```

Comment:
Nit (feel free to disregard): Based on the description (`Minimum work assumed to exist on a valid chain in hex`), "chain" and "value" might be superfluous here
```suggestion
            return util::Error{strprintf(Untranslated("Invalid minimum work specified (%s), must be up to %d characters hex"), *value, uint256::size() * 2)};
```

Reply:
Taken both suggestions in latest force-push, thanks.